test/mbed_gt_tests_spec.py

```diff
@@ -66,11 +66,91 @@
     }
 }
 
+test_case_test_spec = {
+    "builds": {
+        "K64F-ARM": {
+            "platform": "K64F",
+            "toolchain": "ARM",
+            "base_path": "./.build/K64F/ARM",
+            "baud_rate": 115200,
+            "tests": {
+                "tests-mbed_drivers-generic_tests": {
+                    "binaries": [
+                        {
+                            "binary_type": "bootable",
+                            "path": ".build/tests/K64F/ARM/TESTS/mbed_drivers/generic_tests/tests-mbed_drivers-generic_tests.bin"
+                        }
+                    ],
+                    "testcases": [
+                        "Blinky", 
+                        "C++ stack", 
+                        "C++ heap", 
+                        "Basic"
+                    ]
+                },
+                "tests-mbed_drivers-c_strings": {
+                    "binaries": [
+                        {
+                            "binary_type": "bootable",
+                            "path": ".build/tests/K64F/ARM/TESTS/mbed_drivers/c_strings/tests-mbed_drivers-c_strings.bin"
+                        }
+                    ],
+                    "testcases": [
+                        "C strings: strpbrk", 
+                        "C strings: %g %g float formatting", 
+                        "C strings: %f %f float formatting", 
+                        "C strings: %u %d integer formatting", 
+                        "C strings: strtok", 
+                        "C strings: %i %d integer formatting", 
+                        "C strings: %e %E float formatting", 
+                        "C strings: %x %E integer formatting"
+                    ]
+                }
+            }
+        },
+        "K64F-GCC": {
+            "platform": "K64F",
+            "toolchain": "GCC_ARM",
+            "base_path": "./.build/K64F/GCC_ARM",
+            "baud_rate": 9600,
+            "tests": {
+                "tests-mbed_drivers-generic_tests": {
+                    "binaries": [
+                        {
+                            "binary_type": "bootable",
+                            "path": ".build/tests/K64F/GCC_ARM/TESTS/mbed_drivers/generic_tests/tests-mbed_drivers-generic_tests.bin", 
+                            "testcases": [
```

Comment:
You are right this is the so that exact test case in this binary can be stated. For example a ```combined``` binary may have different set of tests. But this is just theoretical assumption.